engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js

```diff
@@ -129,5 +129,9 @@ angular.module('Bastion.content-hosts').controller('ContentHostDetailsInfoContro
 
             return ids.join(" or ");
         };
+	
+        $scope.convertMemToGB = function (memoryValue) {
```

Comment:
@jlsherrill thanks for your help on this one. Based on your suggestions, I have modified the changes and submitted the changes. 

@waldenraines can you please verify the latest commit and let me know, if changes are required.